ozaria/site/components/teacher-dashboard/modals/ModalCurriculumPromotion.vue

```diff
@@ -0,0 +1,112 @@
+<script>
+import ModalDynamicPromotion from './ModalDynamicPromotion'
+import trackable from 'app/components/mixins/trackable.js'
+
+export default Vue.extend({
+  components: {
+    ModalDynamicPromotion
+  },
+  mixins: [trackable],
+  props: {
+    curriculumClicked: {
+      type: Boolean,
+      required: false
+    }
+  },
+  data: () => {
+    return {
+      showModal: true
+    }
+  },
+  computed: {
+    isOld () {
+      const twoDaysAgo = new Date(new Date() - 2 * 24 * 60 * 60 * 1000)
+      return new Date(me.get('dateCreated')) < twoDaysAgo
+    }
+  },
+  watch: {
+    curriculumClicked (newVal, oldVal) {
+      if (newVal) {
+        this.showModal = false
```

Comment:
If we hide the modal this way, the `seen` flag won't be set, and modal will appear again.
Let's call the modal's close method instead:
```suggestion
        this.$refs.modal.close()
```